js/models/Settings.js

```diff
@@ -26,10 +30,40 @@ export default class extends BaseModel {
   nested() {
     return {
       shippingAddresses: ShippingAddresses,
-      smtpSettings: SMTPSettings,
+      appearanceSettings: AppearanceSettings,
+      transactionSettings: TransactionSettings,
+      serverSettings: ServerSettings,
+      smtpIntegrationSettings: SMTPIntegrationSettings
     };
   }
 
+  parse( response ) {
+    const data = this.fromAPIFormatJSON( response );
+    // make it immutable to highlight unintended modification errors
```

Comment:
It was instrumentation and to try to monkey patch the key deletion that was happening with nested models.